openassessment/templates/openassessmentblock/oa_grade_available_responses.html

```diff
@@ -0,0 +1,32 @@
+{% load i18n %}
```

Comment:
Can you create a new folder, `instructor_dashboard`, under `openassessment/templates/openassessmentblock/` and move this file there? That seems to fit with the rest of the template organization a bit more cleanly